esp-hal/src/gpio.rs

```diff
@@ -2735,6 +2739,23 @@ pub mod rtc_io {
                 private: PhantomData,
             }
         }
+
+        #[cfg(esp32c6)]
+        /// Configures the pin as an pullup input and a push pull output pin.
+        pub fn into_puinput_ppoutput(self) -> LowPowerPin<InOut, PIN> {
```

Comment:
that's a weird name - probably `into_open_drain_output` like we do it for HP GPIO is a better name